apps_download/data/cron_scheduler.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright 2013-2017 BizzAppDev - Ruchir Shukla <ruchir@bizzappdev.com>
+    Copyright 2013-2017 AgentERP - Georg Notter <georg.notter@agenterp.com>
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <data>
+        <record forcecreate="True" id="ir_cron_scheduler_generate_zip" model="ir.cron">
```

Comment:
AFAIK `forcecreate` defaults to `True`. Is there a reason this is being defined?